packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -83,16 +90,32 @@ const DMT5TradeModal = ({
                         </Text>
                     )}
                 </div>
-                {mt5_trade_account?.display_balance && (
-                    <Text size='xs' color='profit-success' className='cfd-trade-modal__desc-balance' weight='bold'>
-                        <Money
-                            amount={mt5_trade_account.display_balance}
-                            currency={mt5_trade_account.currency}
-                            has_sign={!!mt5_trade_account.balance && mt5_trade_account.balance < 0}
-                            show_currency
+                <div className='cfd-trade-modal__acc_status'>
+                    {mt5_trade_account?.display_balance && (
+                        <Text size='xs' color='profit-success' className='cfd-trade-modal__desc-balance' weight='bold'>
+                            <Money
+                                amount={mt5_trade_account.display_balance}
+                                currency={mt5_trade_account.currency}
+                                has_sign={!!mt5_trade_account.balance && mt5_trade_account.balance < 0}
+                                show_currency
+                            />
+                        </Text>
+                    )}
+                    {is_open_order_position_status_present && (
```

Comment:
since this is common in trading-app-card and dmt5-trade-modal maybe we can create a common component and move this below functions to that component
 const { is_flag_present: is_open_order_position_status_present, flag_value: open_order_position_status } =
        useIsMt5LoginListStatusPresent('landing_company_short', mt5_trade_account?.login ?? '');
    const status_text = open_order_position_status ? 'No new positions' : 'Account closed';